classes/app.rb

```diff
@@ -66,13 +70,37 @@ def music_albums_navigator(option)
   def games_navigator(option)
     case option
     when '1'
-      Game.list_games
+      @games.each do |game|
+        puts "Date: #{game.id}, \n multiplayer: #{game.multiplayer}"
+        puts "last_played_at: #{game.last_played_at}, \n publish_date: #{game.publish_date}, \n archived: #{game.archived}"
+      end
       app_navigator('3')
+
     when '2'
-      Author.list_authors
+      @authors.each do |author|
+        puts "FirstName: #{author.first_name}, \n LastName: #{author.last_name}"
+      end
       app_navigator('3')
+
     when '3'
-      Game.add_game
+      print('First Name of the author of the game: ')
+      first_name = gets.chomp
+      print('Last Name of the author of the game: ')
+      last_name = gets.chomp
+      @authors << Author.new(first_name, last_name)
+
+      print('Type yes if multiplayer or No if otherwise: ')
+      multiplayer = gets.chomp
+      print('Date played: ')
+      last_played_at = gets.chomp
+      print('Game ID: ')
+      id = gets.chomp.to_i
+      print('Date game was published: ')
+      publish_date = gets.chomp.to_i
+      print('Do you want this game to be archived Yes|No: ')
+      archived = gets.chomp.to_i
+      @games << Game.new(multiplayer, last_played_at, id, publish_date, archived)
+      puts('Author and Game created successfully')
       app_navigator('3')
```

Comment:
Hi, @stephnna :
Thank you for implementing the addition and listing of the game and author class members. We have made good progress so far. However, it would be better to implement these functions _list_games, list_authors_ and _add_games_  as separate functions in the authors and games classes, then call them in the app class in keeping with the structure of the project and SOLID principles.

If you have any questions you can call me on my zoom any time, I'll be happy to discuss